variables.tf

```diff
@@ -85,6 +85,7 @@ variable "sa_org_iam_permissions" {
     "roles/billing.user",
     "roles/compute.networkAdmin",
     "roles/compute.xpnAdmin",
+    "roles/editor",
```

Comment:
What are the org wide editor permissions required for? Managing projects / resources that fall out of the scope of what has been provisioned by CFT?

Reply:
I added this to ensure that the generated service account could subsequently managed further updates to the state created by this module.

Reply:
I see. I'm still not quite sure why editor permissions are required across the org to do that? Can you elaborate?

Reply:
Fair point; it isn't necessarily required across the organization, but applying changes to the bootstrap project using the service account may require the editor role. This can be moved to project scope.

Reply:
Ok cool. The other thing to consider is that the runner of this module, will be project owner for the bootstrap project so will have access to make other changes where required. The service account itself should have storage.admin over the bucket once this PR goes through. (Currently object.admin)

Reply:
Resolved by ee10cb68fef61edf8c303258bea4490b9ec283e2.